docs/core/tla.rst

```diff
@@ -356,20 +356,29 @@ Fairness is formally defined in TLA+ as follows::
 
 In English:
 
-* ``WF_x(A)``: If it is *eventually always* true that the A action *can happen* (in a way that changes v), then it *will* eventually happen (and change v).
-* ``SF_vars(A)``: If it is *always eventually* true that the A action *can happen* (in a way that changes v), then it *will* eventually happen (and change v).
+* ``WF_v(A)``: If it is *eventually always* true that the A action *can happen* (in a way that changes v), then it *will* eventually happen (and change v).
+* ``SF_v(A)``: If it is *always eventually* true that the A action *can happen* (in a way that changes v), then it *will* eventually happen (and change v).
 
 
 Fairness constraints are appended to the definition of ``Spec``. You can see this in the translation of our prior `strong fairness example <strong_fairness_spec>`::
 
-  Spec == /\ Init /\ [][Next]_vars
-          /\ \A self \in Threads : SF_vars(thread(self))
+  Spec == /\ Init /\ [][Next]_v
```

Comment:
I think this should still be `vars` because it's a direct translation from pluscal.